src/script/auth/component/LinkButton.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Wire
+ * Copyright (C) 2019 Wire Swiss GmbH
```

Comment:
```suggestion
 * Copyright (C) 2022 Wire Swiss GmbH
```

Or did this file already exist?

Reply:
no, it's the new one - I'll fix it, thanks